apps/web-giddh/src/app/gst/gstR3/gstR3.component.html

```diff
@@ -78,8 +78,13 @@
                             "
                             aria-controls="dropdown-nested"
                         >
-                            <ng-container>
-                                {{ selectedMonth }}
+                            <ng-container *ngIf="isMonthSelected">
+                                {{ dayjs(selectedMonth).format("MMMM YYYY") }}
```

Comment:
`dayjs` method should not be called in HTML interpolation as it will be called on every change detection.